### ISSUE TITLE
[RFE] Add possibility to scan multiple systems at once (hopefully by adding tab layout for the whole "Scan" screen?)

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/scap-workbench#17]
Description of problem:
scap-workbench supports performing compliance scans of local and remote systems utilizing the `oscap` tool. This is all OK. But since it supports remote scans, why not to allow multiple remote systems to be scanned at once?

Existing scenario:
1) User selects benchmark to scan the system in question against,
2) Then specifies local or remote system to be scanned, clicks "Scan" button

Enhanced scenario:
User applies the same scenario for multiple remote systems (selects appropriate benchmark file and corresponding remote system where the scan should be performed). Clicks "Scan" button. New tab is opened to schedule "new scanning job / task".

Additional information:
The motivation behind this RFE is not to be able to scan thousands of systems at once / in parallel. But hopefully support to scan more than one system at once could be added so people can scan tens of systems at once.

**Comment by matejak** [comment]
This issue is out of scope of the `scap-workbench` use cases.
